Week1/Day2/Exercises XP Gold/Gold_code.py

```diff
@@ -0,0 +1,155 @@
+#Exercise 1: Concatenate Lists
+list1 = [1,2,3,4,5,6,7,8]
+list2 = [1,2,3,4,5,6,7,8]
+list1.extend(list2)
```

Comment:
is there a second way?

Reply:
concatenated_list = [elem for sublist in [list1, list2] for elem in sublist]
print(concatenated_list)